File: Web/src/app/models/answer.ts
import { DatePipe } from "@angular/common";

export class Answer{
  id?:string;
  agree: boolean;
  answerDate: string;
  idUser: string;
  idSurvey:string;
  idQuestion:string;
  public constructor(agree:boolean, answerDate:string,idUser:string,idSurvey:string, idQuestion:string){
    this.agree=agree;
    this.answerDate=answerDate;
    this.idUser=idUser;
    this.idSurvey=idSurvey;
    this.idQuestion=idQuestion;
  }
}

File: ConsentCollector/ConsentCollector.Entities/Consent/Answer.cs
using System;

namespace ConsentCollector.Entities.Consent
{
    public sealed class Answer : Entity
    {
        public Answer(bool agree, DateTime answerDate) : base()
        {
            Agree = agree;
            AnswerDate = answerDate;
        }

        public bool Agree { get; set; }
        public DateTime AnswerDate { get; set; }

        public Guid IdUser { get; set; }

        public Guid IdSurvey { get; set; }

        public Guid IdQuestion { get; set; }

        public User User { get; set; }

        public Survey Survey { get; set; }

        public Question Question { get; set; }
    }
}
